packages/manager/apps/dedicated/client/app/dedicatedCloud/onboarding/constants.js

```diff
@@ -149,18 +152,21 @@ export const US_GUIDES = [
   {
     title: 'dedicated_clouds_onboarding_us_guide1_title',
     description: 'dedicated_clouds_onboarding_us_guide1_description',
+    id: 'How-to-Access-the-NSX-Interface',
     link:
       'https://support.us.ovhcloud.com/hc/en-us/articles/360015961459-How-to-Access-the-NSX-Interface',
   },
   {
     title: 'dedicated_clouds_onboarding_us_guide2_title',
     description: 'dedicated_clouds_onboarding_us_guide2_description',
+    id: 'How-to-Modify-the-vCenter-Access-Policy',
```

Comment:
Related to :
https://github.com/ovh/manager/pull/8968/files#r1116740901

Reply:
It's applied.